cmd/juju/commands/debuglog.go

```diff
@@ -163,16 +163,16 @@ func (c *debugLogCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.Var(cmd.NewAppendStringsValue(&c.params.IncludeLabel), "include-label", "Only show log messages for these logging labels")
 	f.Var(cmd.NewAppendStringsValue(&c.params.ExcludeLabel), "exclude-label", "Do not show log messages for these logging labels")
 
-	f.StringVar(&c.level, "l", "", "Log level to show, one of [TRACE, DEBUG, INFO, WARNING, ERROR]")
+	f.StringVar(&c.level, "l", "", "Log level to show, one of [TRACE, DEBUG, INFO, WARNING, ERROR]. Logs of higher severity (e.g. ERROR) will still be shown when a lower level (e.g. INFO) is selected")
 	f.StringVar(&c.level, "level", "", "")
 
-	f.UintVar(&c.params.Backlog, "n", defaultLineCount, "Show this many of the most recent (possibly filtered) lines, and continue to append")
+	f.UintVar(&c.params.Backlog, "n", defaultLineCount, "Show this many of the most recent (possibly filtered) lines and continue to append new ones")
 	f.UintVar(&c.params.Backlog, "lines", defaultLineCount, "")
-	f.UintVar(&c.params.Limit, "limit", 0, "Exit once this many of the most recent (possibly filtered) lines are shown")
-	f.BoolVar(&c.params.Replay, "replay", false, "Show the entire (possibly filtered) log and continue to append")
+	f.UintVar(&c.params.Limit, "limit", 0, "Show this many of the most recent (possibly filtered) lines and then exits")
+	f.BoolVar(&c.params.Replay, "replay", false, "Show the entire (possibly filtered) log and continue to append new lines")
 
-	f.BoolVar(&c.noTail, "no-tail", false, "Stop after returning existing log messages")
-	f.BoolVar(&c.tail, "tail", false, "Wait for new logs")
+	f.BoolVar(&c.noTail, "no-tail", false, "Show existing log messages and then exits")
+	f.BoolVar(&c.tail, "tail", false, "Show existing log messages and wait for new logs. This is the default behavior")
```

Comment:
```suggestion
	f.BoolVar(&c.tail, "tail", false, "Show existing log messages and wait for new logs (default)")
```

Reply:
Hi Simon, I'm not sure if "(default)" is clear enough for new users, but I'm happy to change it if you strongly believe it should be :)

Reply:
I'm unsure if we even want to point out default behaviour, but others might say otherwise.

Reply:
We generally don't point out default behaviour anywhere else so I don't think we want to do that here. We can be sure to mention it in the main help text.
Also, to be consistent with the other arg description text, this should be
"Show existing log messages and continue to append new ones"

Reply:
Sure will refactor this thanks Simon and Ian!